file_test.go

```diff
@@ -307,7 +307,7 @@ func TestExtractFiles(t *testing.T) {
 		}
 		defer os.RemoveAll(tempDir)
 
-		err = extractFiles(tc.localFilePath, tc.filePathToExtract, tempDir)
+		_, err = extractFiles(tc.localFilePath, tc.filePathToExtract, tempDir)
```

Comment:
This test should probably check that `extractFiles` is returning the expected number of files.

Reply:
Ah, and you nicely provided the # of files. Updated the code, it works against the public repos and by extension I assume it will work against the private repos that I don't have access to.